### ISSUE TITLE
Possible new Resource

### ISSUE THREAD

**Issue by MykaelRay** [jonathan-bower/DataScienceResources#3]
In this write up, I saw that you have a section for getting started in data science, and I want to show you a new article that was just recently published that would fit in extremely well in that section. It is a Q&A session with the University of Wisconsin's data science faculty, and the answers that they provide make it the perfect resource for ANYONE who is new to the industry, or looking to get into it. Take a look for yourself, I'm sure that you will agree with me. 

http://datasciencedegree.wisconsin.edu/blog/qa-with-the-uw-data-science-faculty/ 

Let me know your thoughts about this idea, I would love to discuss it with you. 

Have a great day, 
Mykael Ray

**Comment by jaytaylopub** [comment]
Yeah, that would be a good one for this area of the page: https://github.com/jonathan-bower/DataScienceResources#data-science-courses

@jonathan-bower You updating?

**Comment by dealsinfotech** [comment]
Well in that section. It is a Q&A session with the University of Wisconsin's data science faculty, and the answers that they provide make it the perfect resource for ANYONE, must [suggest](https://dealsinfotech.com/hotstar-subscription-offer/) to my fellow ones
